docs/content/core/volumes.md

```diff
@@ -27,17 +27,17 @@ If you are familiar with Docker Compose, then you must have attached files or di
       - /Users/alex/mysite:/var/www:rw
 ```
 
-It is an unnamed volume with bind driver. However, to support various OS in Docksal, we cannot always rely on the 
-bind driver and should not use unnamed volumes. 
+That is a called a bind mount - mounting a files/folder on the host directly into a container. 
```

Comment:
Let's return to what it was here. First the change didn't bring any new info. Second I deliberately avoided calling attaching volumes "mounting" to avoid confusing people who are new that everything is mounting. Mounting is used to describe a process of accessing host files from VM.